sdks/python/apache_beam/io/filesystem.py

```diff
@@ -945,3 +945,6 @@ def report_lineage(self, path, unused_lineage, level=None):
     Unless override by FileSystem implementations, default to no-op.
     """
     pass
+
+  def check_splittability(self, path):
+    return True
```

Comment:
This should probably not always be true. If this is a default, perhaps it should not have a default but be abstract and we implement for various filesystems. If it is the default, comment so we understand that is why it ignores the argument.